it/content/src/main/content/jcr_root/content/forms/af/core-components-it/samples/textinput/basic/.content.xml

```diff
@@ -213,6 +213,23 @@
                         validationStatus="valid"/>
                 <fd:events jcr:primaryType="nt:unstructured"/>
             </textinput_1477533951>
+            <textinput_278740740
```

Comment:
Test case is faliing since you added a new component to an existing collateral

Reply:
Updated test collateral